lib/over_react.dart

```diff
@@ -27,7 +27,7 @@ export 'package:react/react.dart' show
     SyntheticUIEvent,
     SyntheticWheelEvent;
 
-export 'package:react/react_client.dart' show setClientConfiguration, ReactElement;
+export 'package:react/react_client.dart' show setClientConfiguration, ReactElement, ReactComponentFactoryProxy;
```

Comment:
This is a bonus change that I'm including in this PR. In 2.0.0-dev, a breaking change to the type of `UiProps.componentFactory` from `Function` to the more specific type of `ReactComponentFactoryProxy`. To help with the transition, we should export this symbol on 1.x as well so that consumers get valid overrides when using the builder's generated outputs on a Dart1/over_react1.